autoload/clap/provider/filer.vim

```diff
@@ -173,22 +164,16 @@ function! s:tab_action() abort
     return
   endif
 
-  if has_key(s:filer_empty_cache, s:current_dir)
-    if g:clap.display.get_lines() != [s:DIRECTORY_IS_EMPTY]
-      call g:clap.display.set_lines([s:DIRECTORY_IS_EMPTY])
```

Comment:
I think it's still useful to indicate that directory is empty even you inject the faked query candidate.